helpers/update-site/documentation.js

```diff
@@ -437,16 +469,29 @@ const getHintFileInfo = async (file) => {
      *
      * e.g.    this is a "description" => this is a \"description\"
      */
-    let description = metaDescription ? metaDescription[1].trim().replace(/\\`/g, '`')
-        .replace(/([^\\])"/g, '$1\\"') : (getPredefinedDescription(file.dest) || getDescription(file.content));
+    let description;
 
-    /*
-     * Remove the character the quotes from the text (' ` ") and the character ',' if it is the last character.
-     * e.g: 'Runs axe-core tests in the target', -> Runs axe-core tests in the target
-     */
-    if (meta) {
-        description = description[description.length - 1] === ',' ? description.substr(1, description.length - 3) : description.substr(1, description.length - 2);
+    if (metaDescription) {
+        description = metaDescription[1].trim().replace(/\\`/g, '`')
+            .replace(/([^\\])"/g, '$1\\"');
+
+        /*
+         * Remove the character the quotes from the text (' ` ") and the character ',' if it is the last character.
+         * e.g: 'Runs axe-core tests in the target', -> Runs axe-core tests in the target
+         */
+        if (!description.startsWith('getMessage')) {
+            description = description[description.length - 1] === ',' ? description.substr(1, description.length - 3) : description.substr(1, description.length - 2);
+        }
+    } else {
+        description = getPredefinedDescription(file.dest) || getDescription(file.content);
     }
+
+    if (description.startsWith('getMessage')) {
+        description = getDescriptionFromLocalization(file).trim()
+            .replace(/\\`/g, '`')
```

Comment:
Is this necessary? All of the localization files are JSON so strings should be wrapped in double-quotes. Not sure why we'd need to remove an escape in front of a back-tick.

Reply:
Take a look to the comment in line 461. It looks like it is a `hexo` requirement hehehe.

Reply:
I thought that was just because we had used a back-tick as the delimiter for a number of our descriptions in `meta.ts` files in the past. Are you sure it's a `hexo` requirement?

Reply:
Ok, we found the other comment that suggests this is needed for `hexo`.